public/app/plugins/datasource/loki/query_utils.test.ts

```diff
@@ -1,56 +1,87 @@
-import { parseQuery } from './query_utils';
+import { parseQuery, getHighlighterExpressionsFromQuery } from './query_utils';
+import { LokiExpression } from './types';
 
 describe('parseQuery', () => {
   it('returns empty for empty string', () => {
     expect(parseQuery('')).toEqual({
       query: '',
       regexp: '',
-    });
+    } as LokiExpression);
```

Comment:
Is the `as LokiExpression` really needed? (used in the whole file)

Reply:
I wanted to have type validation on the output of the function. Is there a different way? Or is this pointless?